.github/ISSUE_TEMPLATE/flaking-test.yaml

```diff
@@ -39,10 +39,10 @@ body:
     attributes:
       label: Anything else we need to know?
 
- - type: textarea
-    id: templateLabel
-    attributes:
-      label: Label(s) to be applied
-      value: |
-        /kind flake
-        One or more /area label. See https://github.com/metal3-io/cluster-api-provider-metal3/labels for the list of labels.
\ No newline at end of file
+  - type: textarea
+      id: templateLabel
```

Comment:
there seems to be still indentation issue, basically `id`and whole thing belopw should be aligned on the same line with `type`

Reply:
Nice catch